# Germline variant discovery for rare disease diagnostics

| Audience | Computational skills required| Duration |
:----------|:----------|:----------|
| Biologists | Introduction to the Command-line | 3 - 2.5 hour sessions |

### Description

This ***Introduction to Variant Analysis*** workshop is aimed at providing best practices for calling variants ...

This workshop series should provide participants with an ability to take raw sequence reads and process them into a VCF file with annotated variants. Furthermore, the workshop series ends with a tutorial for visualizing called variants within the Integrative Genomics Viewer (IGV). 

> These materials were developed for a trainer-led workshop, but are also amenable to self-guided learning.

#### Pre-Requisite

This workshop series is designed for users with a background in the fundamentals of working in the shell environment on an high-performance computing cluster (HPCC). If you don't have this background or are unsure if you are ready for this workshop series, you should first complete our [***Introduction to the Command Line***](https://hbctraining.github.io/Intro-to-shell-flipped/) workshop series.

### Learning Objectives

* ..
* .

### Lessons
* [Workshop schedule]()
* [Self-learning]()

### Installation Requirements
***All:***
- [FileZilla Client](https://filezilla-project.org/download.php?type=client) (make sure you get 'FileZilla Client')
- [Integrative Genomics Viewer (IGV)](https://software.broadinstitute.org/software/igv/)

***Mac users:***
No additional installation requirements.  

***Windows users:***
[GitBash](https://git-scm.com/download/win)  


****

*These materials have been developed by members of the teaching team at the [Harvard Chan Bioinformatics Core (HBC)](http://bioinformatics.sph.harvard.edu/). These are open access materials distributed under the terms of the [Creative Commons Attribution license](https://creativecommons.org/licenses/by/4.0/) (CC BY 4.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.*
